project_event/models/project_task.py

```diff
@@ -862,8 +863,9 @@ def action_draft(self):
     @api.multi
     def action_postpone(self):
         if self.is_type_task() and self.task_state in \
-                ['requested', 'read', 'canceled', 'accepted']:
-            self.draft_resources_reservation()
+                ['requested', 'read', 'canceled', 'accepted', 'option']:
```

Comment:
@wbeverly  is this correct?

Reply:
@rimbendhaou @wbeverly I remember that the changes of the state were right, we tested them,  but maybe we missed this. Also, IMHO  a resource that is in state 'option', it doesn't make sense to postpone its event.

Reply:
You can refer to the document I left with rim, and also on : 
https://projects-odoo.savoirfairelinux.com/documents/23

You should have the possibility to 'REPORTER' an EV in OPTION, APPROVÉ
You should have the possibility to 'REPORTER' an AC in OPTION, APPROVÉ
You should have the possibility to 'REPORTER' an TA in OPTION, DEMANDÉ, LU, ACCEPTÉ

Reply:
Right, but this condition if for managing messages